lib/home/profile/account/account_field.dart

```diff
@@ -70,7 +70,7 @@ class _AccountFieldState extends State<AccountField> {
 
   @override
   Widget build(BuildContext context) {
-    final colors = context.colorScheme;
+    final colors = ThcColors.of(context);
```

Comment:
Instead of making an `extension on BuildContext`, I thought it'd be nice to follow the conventional Flutter syntax.